lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module Capybara
+        # Checks for there is a more specific matcher offered by Capybara.
+        #
+        # @example
+        #
+        #   # bad
+        #   expect(page).to have_selector('button')
+        #   expect(page).to have_none_of_selectors('button.cls')
```

Comment:
Leftover?

Reply:
Or the "good" example `expect(page).to have_button(class: 'cls')` probably needs to be changed.

Reply:
Hmm, it seems `have_none_of_selectors` causing offense is not covered by specs. Perhaps it should be?

Reply:
We decided not to deal with `have_any_of_selectors`/`have_none_of_selectors`/`have_all_of_selectors`, as they accept several locators, while those matchers that we currently inspect accept just one.
This can be quite hard to do right. Related discussion: https://github.com/rubocop/rubocop-rspec/pull/1294#discussion_r890823552

The cop might see an improvement on this front, and e.g. autocorrection in a follow-up.

Reply:
Oops, sorry. I fixed it.